package-lock.json

```diff
@@ -5522,7 +5522,7 @@
         },
         "readable-stream": {
           "version": "1.0.34",
-          "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
```

Comment:
Is there a reason for it to be http and not https?

Reply:
Not sure, that's how npm generated the file

Reply:
Interesting, why for this package only

Reply:
That's a riddle
I presume it should be `https`, version bump shouldn't update any URLs

Reply:
Version bump regenerates the lock file anyway.

Here is what I found
https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818

Reply:
Will try to remove `node_modules` and run install one more time

Reply:
If it doesn't help we might need to bring it back to `https` manually

Reply:
Don't think it has any sense, lock files shouldn't be touched manually. I think that the problem is, I'm using `node@16` that uses `npm` that generates lock version `2`, so I installed deps, then found that the lock file is completely changed, reverted it, switched to `node@14`, and changed `package.json` version. I think these steps affected the resulting lock file after version change.

Ok, removed `node_modules`, ran `npm install`, and the problem went away

Reply:
Looks like `npm@7` resolves the package verion to `http` while `npm@6` doesn't

Reply:
@wontem will take it into account when will be switching versions, thank you for investigating this